frappe/database/db_manager.py

```diff
@@ -48,7 +48,7 @@ def grant_all_privileges(self, target, user, host=None):
 		if not host:
 			host = self.get_current_host()
 
-		self.db.sql("GRANT ALL PRIVILEGES ON `%s`.* TO '%s'@'%s';" % (target, user, host))
+		self.db.sql("GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, INDEX, ALTER, CREATE TEMPORARY TABLES, CREATE VIEW, EVENT, TRIGGER, SHOW VIEW, CREATE ROUTINE, ALTER ROUTINE, EXECUTE ON `%s`.* TO '%s'@'%s';" % (target, user, host))
```

Comment:
maybe let's not change the default line, lets add a flag that does this. something like enable_rds inside site_config or common_site_config would do.

it wouldn't make much of a difference, but let's keep things more specific to services :)

Reply:
i suppose that will need more complex changes, can't do it. will close this pull request.

Reply:
shouldn't take a lot, you just need to read `frappe.conf` for a key, suppose `rds_db` and set its value to `1` or something.

Reply:
ok